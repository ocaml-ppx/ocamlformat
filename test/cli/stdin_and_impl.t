  $ echo 'let x =       1' | ocamlformat --impl -
  let x = 1
