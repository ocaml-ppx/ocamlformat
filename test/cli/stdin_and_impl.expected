let () = print_endline A.x
